gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Oct 02 20:56:36 PDT 2017
+#Thu Oct 12 21:29:57 PDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```

Comment:
Same here, with that version the project doesn't work on Android Studio 2.3.3 (I also changed that line in the downgrade)

Reply:
same won't checking in future